src/test/kotlin/no/nav/helse/fritakagp/web/api/resreq/validation/AaregConstraintsKtTest.kt

```diff
@@ -34,18 +34,6 @@ class AaregConstraintsKtTest {
         val arbeidsforhold = objectMapper.readValue<List<Arbeidsforhold>>(aaregFile)
             // Legg inn organisasjonsnummer
             .filter { it.arbeidsgiver.organisasjonsnummer == "XXXXXXXX" }
-
-        // Endre til perioden kravet gjelder
-        val arbeidsgiverPeriode = Arbeidsgiverperiode(
```

Comment:
Hvorfor er dette fjernet? 
Testen er bare brukt for å feilsøke prodfeil i aareg sjekken.
Er lenge siden vi har måtte gjøre det. Men synes det er greit å ha den der hvis vi må gjøre det igjen.

Reply:
Yes, my bad. Dette funka ikke på min lokale maskin, så trodde det var derfor at testen var disabled. 
Lagt det tilbake nå.